src/main/java/tech/aroma/data/cassandra/Mappers.java

```diff
@@ -519,6 +549,53 @@ else if (doesRowContainColumn(row, Tables.Users.EMAILS))
         };
     }
     
+    
+    //==========================================================
+    // USER PREFERENCES MAPPER
+    //==========================================================
+    
+    /**
+     * Extracts {@linkplain MobileDevice Mobile Devices} from a {@linkplain Tables.UserPreferences User Preferences Row}.
+     * 
+     * @return 
+     */
+    static Function<Row, Set<MobileDevice>> mobileDeviceMapper()
+    {
+        return row ->
```

Comment:
We might want to start adding null checks to rows, in case they do come back null for whatever reason.